projects/online/online/main.py

```diff
@@ -274,12 +288,88 @@ def main(
     highpass: Optional[float] = None,
     refractory_period: float = 8,
     far_threshold: float = 1,
-    server: str = "test",
+    server: GdbServer = "test",
     ifo_suffix: str = None,
     input_buffer_length: int = 75,
     output_buffer_length: int = 8,
+    samples_per_event: int = 20000,
+    nside: int = 32,
     device: str = "cpu",
 ):
+    """
+    Main function for launching real-time Aframe and AMPLFI pipeline.
+
+    Args:
+        aframe_weights:
+            Path to trained Aframe model weights
+        amplfi_architecture:
+            AMPLFI model architecture for parameter estimation
+        amplfi_weights:
+            Path to trained AMPLFI model weights
+        background_path:
+            Path to background noise events dataset
+            used for Aframe FAR calculation
+        foreground_path:
+            Path to recovered injection events dataset
+        rejected_path:
+            Path to rejected injection parameters dataset
+        outdir:
+            Directory to save output files
+        datadir:
+            Directory containing input strain data
+        ifos:
+            List of interferometer names
+        inference_params:
+            List of parameters on which the AMPLFI
+            model was trained to perform inference
+        channels:
+            List of the channel names to analyze
+        sample_rate:
+            Input data sample rate in Hz
+        kernel_length:
+            Length of Aframe analysis kernel in seconds
+        inference_sampling_rate:
+            Rate at which to sample the output of the Aframe model
+        psd_length:
+            Length of PSD estimation window in seconds
+        aframe_right_pad:
+            Time offset for trigger positioning in seconds
+        amplfi_kernel_length:
+            Length of AMPLFI analysis window in seconds
+        event_position:
+            Event position (in seconds) from the left edge
+            of the analysis window used for parameter estimation
+        fduration:
+            Length of whitening filter in seconds
+        integration_window_length:
+            Length of output integration window in seconds
+        astro_event_rate:
+            Expected rate of astrophysical events in Hz
+        fftlength:
+            FFT length in seconds (defaults to kernel_length + fduration)
+        highpass:
+            High-pass filter frequency in Hz
+        refractory_period:
+            Minimum time between events in seconds
+        far_threshold:
+            False alarm rate threshold in events/day
+        server:
+            GraceDB server to use ("test" or "production")
+        ifo_suffix:
+            Optional suffix for IFO channel names
```

Comment:
The suffix is for the directory name, not the channel name. Live data is just in "H1", while replay data is in "H1_{suffix}"

Reply:
The variable could be named better